frontend/src/components/OrderDetails/index.tsx

```diff
@@ -207,8 +207,10 @@ const OrderDetails = ({
         amount: amountString,
         method: order.payment_method,
       });
-      receive = t('You receive via Lightning {{amount}} Sats (Approx)', {
+      const receives_on_chain = order.swap_fee_rate > 0;
```

Comment:
Is `order.swap_fee_rate` the best param we can use for this? This might display "onchain" for everyone while they are in the step of submitting an invoice.

Ideally we have a `order.is_swap` boolean that is only sent to the buyer.

Reply:
Since we don't have this param from the API now, i just changed the text.